configure.ac

```
@@ -1635,6 +1635,12 @@ SQUID_CHECK_LIB_WORKS(ldap,[
     AC_CHECK_HEADERS(mozldap/ldap.h)
     SQUID_CHECK_LDAP_API
   ])
+  # We can still rely on deprecated APIs on MacOS
+])
+AS_IF([test "x$squid_host_os.$ac_cv_header_ldap_h.$ac_cv_lib_ldap_ldap_init.$squid_cv_LDAP" = "xdarwin.yes.yes.1"],[
```

SQUID_CHECK_LIB_WORKS() API documents that the macro sets LIBFOO_LIBS on success. Do we have to test some obscure internal variables instead?!

```suggestion
AS_IF([test "x$squid_host_os" = "xdarwin" -a "x$LIBLDAP_LIBS" != "x"],[
```